Recipes/Recipes/ProfileModule/ViewControllers/BonusesViewController.swift

```diff
@@ -0,0 +1,141 @@
+// BonusesViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+private enum Constants {
+    static let fontVerdana = "Verdana"
+    static let fontVerdanaBold = "Verdana-Bold"
+    static let bonusesImage = "bonusesImage"
+    static let xmarkImage = "xmark"
+    static let yourBonusesText = "Your bonuses"
+    static let countBonusText = "⭐️ 100"
+    static let leftInset = UIScreen.main.bounds.width / 2 - 80
+    static let termsTitleText = "Terms of Use"
+    static let termsKeyText = "terms_use"
+}
+
+/// Информация о бонусах
+final class BonusesViewController: UIViewController {
+    // MARK: - Static Constant
+
+    static let reuseID = String(describing: PersonalInfoCell.self)
```

Comment:
```suggestion
    static let reuseID = String(describing: PersonalInfoCell.self)
```
это в самих ячейках делай